src/gui/editors/PianoRoll.cpp

```diff
@@ -1704,6 +1704,7 @@ void PianoRoll::mousePressEvent(QMouseEvent * me )
 			if( me->buttons() == Qt::RightButton )
 			{
 				// right click, tone marker contextual menu
+				m_pianoKeySelected = getKey( me->y() );
```

Comment:
```suggestion
				m_pianoKeySelected = getKey(me->y());
```

Reply:
Yes but the surrounding code is the older style. It doesn't look good when we mix the styles. I'm waiting for some automatic code formatting to come true. Cool?